packages/compiler-cli/ngcc/src/execution/cluster/worker.ts

```
@@ -29,16 +29,16 @@ export async function startWorker(logger: Logger, createCompileFn: CreateCompile
       (_task, outcome, message) => sendMessageToMaster({type: 'task-completed', outcome, message}));
 
   // Listen for `ProcessTaskMessage`s and process tasks.
-  cluster.worker.on('message', async (msg: MessageToWorker) => {
+  cluster.worker?.on('message', async (msg: MessageToWorker) => {
```

I'd rather throw an error if `cluster.worker` is undefined, as otherwise this will silently cause workers to do nothing at all without errors.

Given that `cluster.worker` is used in the listener callback as well, I propose assigning it to a local variable and throwing an error if that's undefined, then you don't need to account for an undefined value anymore in the closure.